ga4gh/backend.py

```diff
@@ -552,6 +552,48 @@ def variantAnnotationsGenerator(self, request):
             request, variantAnnotationSet)
         return intervalIterator
 
+    def featuresGenerator(self, request):
+        """
+        :param request: JSON string - the original web request
```

Comment:
Might as well follow the standard pattern for this comment. Sphinx docs are good, but we should try to apply them systematically, file wide if they are introduced.